admin/client/dist/js/MemberDatetimeOptionsetField.js

```diff
@@ -32,9 +32,9 @@
 
 		$('.memberdatetimeoptionset .toggle').entwine({
 			onclick: function onclick(e) {
-				jQuery(this).closest('.form__field-description').parent().find('.toggle-content').toggle();
+				jQuery(this).closest('.description').parent().find('.toggle-content').toggle();
```

Comment:
That's a good point, what do you guys think about minifying the dist files?

I would have committed into legacy but assumed from its folder name `legacy` that it wasn't relevant any more. A minified file would be impossible to confuse :-)